src/robot_interface.cpp

```diff
@@ -0,0 +1,52 @@
+/*!
+ * @file robot_interface.cpp
```

Comment:
This class is mostly accessors and mutators.  Consider consolidating these portions of the implementation into `inline` functions in the class definition in `robot_interface.h`.